data-model/fdo-type/annotation/0.3.0/examples/tombstoned-annotation.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@id": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+  "@type": "ods:Annotation",
+  "ods:ID": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+  "rdf:type": "ods:Annotation",
+  "ods:status": "ods:Tombstone",
+  "ods:version": 1,
+  "oa:motivation": "oa:commenting",
+  "oa:hasTarget": {
+    "@id": "https://hdl.handle.net/20.5000.1025/XYZ-XYZ-XYZ",
+    "@type": "ods:DigitalSpecimen",
+    "ods:ID": "https://hdl.handle.net/20.5000.1025/XYZ-XYZ-XYZ",
+    "ods:type": "https://doi.org/21.T11148/bbad8c4e101e8af01115",
+    "oa:hasSelector": {
+      "@type": "ods:FieldSelector",
+      "ods:field": "$.hasEvent[0].location.dwc:country"
+    }
+  },
+  "oa:hasBody": {
+    "@type": "oa:TextualBody",
+    "oa:value": [
+      "Venezuela"
+    ],
+    "dcterms:references": "https://api.gbif.org/v1/occurrence/search?occurrenceID=https://herbarium.bgbm.org/object/BW00965020&catalogNumber=B -W 00965 -02 0&basisOfRecord=PreservedSpecimen",
+    "ods:score": 0.9
+  },
+  "dcterms:creator": {
+    "@id": "https://orcid.org/0000-0002-5669-2769",
+    "@type": "schema:Person",
+    "schema:name": "John Doe"
+  },
+  "dcterms:created": "2024-06-27T12:00:00Z",
+  "as:generator": {
+    "@id": "https://hdl.handle.net/20.5000.1025/ABC-DEF-GHI",
+    "@type": "as:Application",
+    "schema:name": "Annotation Generator Service"
+  },
+  "dcterms:issued": "2024-06-27T12:00:00Z",
+  "ods:TombstoneMetadata": {
+    "@type": "ods:Tombstone",
+    "ods:tombstonedDate": "2023-06-15T14:30:00Z",
+    "ods:tombstonedReason": "This object was made by mistake",
+    "ods:TombstonedByAgent": {
```

Comment:
TombstonedByAgent should lower case? -> tombstonedByAgent?

Reply:
Yes that is what we discussed in as namingconvention, if we create new terms and they contain an object we capitalize them to indicate that it contains an object, in this case an agent.